docs/guides/transfer_learning.md

```diff
@@ -132,49 +144,52 @@ If you need to to train different parts of the model separately, you have two op
 2. Filter the parameters for `jax.grad`.
 3. Use multiple optimizers for different parameters.
 
-While each could be useful in different situations, its recommended to use use multiple optimizers via [Optax](https://optax.readthedocs.io/)'s [`optax.multi_transform`](https://optax.readthedocs.io/en/latest/api.html#optax.multi_transform) because it is efficient and can be easily extended to implement differential learning rates. To use `optax.multi_transform` you have to do two things:
+For most situations we will recommend to use multiple optimizers via [Optax](https://optax.readthedocs.io/)'s [`multi_transform`](https://optax.readthedocs.io/en/latest/api.html#optax.multi_transform) as its both efficient and can be easily extended to implement many fine-tunning strategies. 
 
-1. Define some parameter partitions.
-2. Create a mapping between partitions and their optimizer.
-3. Create a pytree with the same shape as the parameters but its leaves containing the corresponding partition label.
+### **optax.multi_transform**
 
-## Freeze layers
+To use `optax.multi_transform` following must be defined:
+
+1. The parameter partitions.
+2. A mapping between partitions and their optimizer.
+3. A pytree with the same shape as the parameters but its leaves containing the corresponding partition label.
 
-To freeze layers with `optax.multi_transform`, create the `trainable` and `frozen` parameter partitions.
+## Freeze layers
```

Comment:
Do you need a new section here? Seems this is also related to the optimizer and the previous section is very short. I would recommend just removing this heading.